README.md

```diff
@@ -15,7 +20,7 @@ Return a list of view names.
 _Example request_
 
 ```sh
-$ curl -X GET http://relation_engine/api/views
+$ curl -X GET http://{root_url}/api/views
```

Comment:
Should probably remove http:// from all of these as it's in the root url above

Reply:
:+1: